README.md

```diff
@@ -1,315 +1,248 @@
-```
-
-./mvnw
-
-```
+# OncoKB Curation
+
+## Local Development
+
+1. Before you can build this project, you must install the following
+   dependencies on your machine:
+
+   1. [Docker](https://www.docker.com/get-started).
+   2. [mysql](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/).
+   3. [Java 11](https://dev.java/download/) SDK.
+   4. [Node.js](https://nodejs.org): We use Node to run a development web server
+      and build the project. Depending on your system, you can install Node either
+      from source or as a pre-packaged bundle. The version of node we are using can
+      be found in the [.nvmrc](.nvmrc) file.
+
+      - If you are using [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating)
+        simply run the following commands from the root directory of the project
+        to install and use the correct version of node.
+
+      ```sh
+      nvm install
+      nvm use
+      ```
+
+   5. [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/)
+
+2. After installing Node and yarn, you should be able to run the following command
+   to install development tools. You will only need to run this command when
+   dependencies change in [package.json](package.json).
+
+   ```sh
+   yarn install
+   ```
+
+   We use yarn scripts and [Webpack](https://webpack.js.org/) as our build system.
+
+3. If you are using redis as a cache, you will have to launch a cache server.
+
+   - To start your cache server, run:
+
+     ```sh
+     docker-compose -f src/main/docker/redis.yml up
+     ```
+
+   - The cache can also be turned off by adding to the [application-dev.yaml](src/main/resources/config/application-dev.yml):
+
+     ```yaml
+     spring:
+       cache:
+         type: none
+     ```
+
+   See [here](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-caching.html#boot-features-caching-provider-none)
+   for details.
+
+4. Configure Keycloak and Google SSO
+
+   To log in to your app, you'll need to have [Keycloak](https://keycloak.org)
+   up and running.
+
+   ```sh
+   docker-compose -f src/main/docker/keycloak.yml up
+   ```
+
+   1. Once keycloak server is running, go to `http://localhost:8080` and click
+      `Administration Console`
+   2. Login with the credential `username: admin, password:admin`.
+
+      - Note this will match the credentials in the [keycloak.yml](src/main/docker/keycloak.yml).
+
+   3. [Create a realm](https://www.keycloak.org/docs/latest/server_admin/#proc-creating-a-realm_server_administration_guide)
+      and name it `oncokb-curation`
+   4. [Create a new openid-connect client](https://www.keycloak.org/docs/latest/server_admin/#assembly-managing-clients_server_administration_guide)
+      - Set Access Type to `confidential`
+      - Enable Standard Flow
+      - Enable implicit flow
+      - Disable access grants
+      - Disable service accounts
+      - Disable OAuth 2.0 Device Authorization Grant
+      - Disable Authorization
```

Comment:
Are these all additional steps we have to take? I don't recall I did anything above except specially enable Auth. Has this changed in K18?